plugins/module_utils/controller.py

```diff
@@ -184,6 +184,23 @@ def create_if_needed(
         msg = f"Unable to create {item_type} {item_name}: {response.status}"
         raise EDAError(msg)
 
+    def create(
+        self, new_item: dict[str, Any], endpoint: str, item_type: str = "unknown"
+    ) -> dict[str, bool]:
+        """Run a create (post) operation unconditionally and return the result."""
+
+        response = self.post_endpoint(endpoint, data=new_item)
+
+        if response.status in [201, 202]:
+            self.result["changed"] = True
+            return self.result
+
+        if response.json:
+            error_msg = f"Unable to create {item_type} {new_item}: {response.json}"
+
+        error_msg = f"Unable to create {item_type} {new_item}: {response.status} {response.data}"
```

Comment:
¿If there is a json decodable response set error_msg but then replace it with a different error_msg regardless?

Reply:
good catch, thanks. Fixed